nf_core/modules/lint/module_tests.py

```diff
@@ -40,6 +40,18 @@ def module_tests(_, module):
         module.failed.append(("test_main_exists", "test `main.nf` does not exist", module.test_main_nf))
         module.warned.append(("test_main_exists", "test `main.nf.test` does not exist", nftest_main_nf))
 
+    if os.path.exists(nftest_main_nf):
+        # Check if main.nf.test.snap file exists, if 'snap(' is inside main.nf.test
+        with open(nftest_main_nf, "r") as fh:
+            if "snapshot(" in fh.read():
```

Comment:
Maybe we should also account for people who don't type their bracket immediately after snapshot (`snapshot (` for example)

Reply:
I think we can be a bit opinionated about the formatting here.

Reply:
True, I'm just thinking about what could go wrong. but if we enforce `snapshot(` then it should be fine